lint-staged.config.mjs

```diff
@@ -1,5 +1,5 @@
 const config = {
-  '*.{js,jsx,ts,tsx}': ['eslint --fix --max-warnings 100', 'prettier --write'],
+  '*.{js,jsx,ts,tsx}': ['eslint --fix --max-warnings 30', 'prettier --write'],
```

Comment:
@Dzmitry-Yaniuk 
What do you think about decreasing warnings to '0'? It was '0' before switching to nextjs, so I guess we can try to continue doing that.

Reply:
Good idea! From opinion absence of warnings in code is a project good shape indicator.
However, with existing applied rules we get already around 10 warnings, refactoring code in terms of this PR might not be a good idea. Perhaps I can set this limit now to 10 items?
Otherwise code cannot be pushed without bypass option for git commit

Reply:
Totally agree, so let's set the lowest available number for now and I will create additional ticket for reaching 0